### PR TITLE
refactor: ziofy ValuesResponderV2 (DEV-1757)

### PR DESCRIPTION
**Pull request by mpro7**
### Task Description/Number

Issue Number: DEV-1757

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/v2/ValuesResponderV2.scala [hunk 1]
```diff
@@ -36,20 +40,29 @@ import org.knora.webapi.messages.v2.responder.resourcemessages._
...
 
 /**
  * Handles requests to read and write Knora values.
  */
-class ValuesResponderV2(
-  responderData: ResponderData,
-  implicit val runtime: zio.Runtime[ResourceUtilV2 with PermissionUtilADM]
-) extends Responder(responderData.actorDeps) {
+trait ValuesResponderV2
+final case class ValuesResponderV2Live(
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/responders/v2/ValuesResponderV2.scala:55]: can this be right that there is nothing in the trait?

Comment (BalduinLandolt) [reply webapi/src/main/scala/org/knora/webapi/responders/v2/ValuesResponderV2.scala:55]: and shouldn't the `ValuesResponderV2Live` extend the trait?

**Review by BalduinLandolt — COMMENTED** [review]
see my comment on the ValuesResponderV2 trait

**Review by seakayone — CHANGES_REQUESTED** [review]
Please fix tests first.

**Pull request closed without merging**